src/decisionengine/framework/engine/Workers.py

```diff
@@ -54,11 +55,11 @@ def run(self):
             os.path.dirname(self.logger_config["log_file"]), myname + ".log"
         )
 
-        logger = structlog.getLogger(f"{myname}")
+        self.logger = structlog.getLogger(CHANNELLOGGERNAME)
 
         # setting a default value here. value from config file is set in call
         # self.task_manager.set_loglevel_value after logger configuration is completed
-        logger.setLevel(logging.DEBUG)
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
This will always give an error (`structlog.Logger` has no method `setLevel()`), you must use the std logger w/ the same name: 
```
logging.getLogger(CHANNELLOGGERNAME).setLevel(logging.DEBUG)
```

This will affect the level also of the structlog logger

Reply:
OK. Never saw an error message but I made the change.